themes/courses.labster.com/lms/templates/labster_notification.html

```diff
@@ -7,7 +7,7 @@
   <div class="msg">
     <i class="msg-icon fa fa-warning"></i>
     <div class="msg-content">
-      <h3 class="title">${_("New Labster.com!")}</h3>
+      <h3 class="title">${_("Welcome!")}</h3>
```

Comment:
why we need change this one ?
also 
fyi: when I check in the html I found a dead link, see, https://github.com/Livit/Livit.Learn.EdX/blob/40811ad8b1e275d7eeb9af8402476e276610e410/themes/courses.labster.com/lms/templates/labster_notification.html#L15

should we change this to the appropriate  url ?

Reply:
> why we need change this one ?

Reverted, We might want to use this modal in the future to inform our user about upcoming maintenance or something like that. Just need to update the code to be more useful and flexible.